modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenExchangeGrantTypeTestCase.java

```diff
@@ -256,7 +521,10 @@ private void createServiceProviderInSecondaryIS() throws Exception {
 
             }
         }
-        Assert.assertTrue(success, "Failed to update service provider with inbound OIDC configs in secondary IS");
+
+        if(!success) {
```

Comment:
fix formatting

Reply:
addressed with [e22de73](https://github.com/wso2/product-is/pull/17605/commits/e22de736ffc31a4350459dbfb3151c8c1f150116)